packages/project-access/src/i18n.ts

```diff
@@ -0,0 +1,35 @@
+import type { TOptions } from 'i18next';
```

Comment:
What does this have to do with a module called `project-access`?

Reply:
I think you misunderstood my comment. I meant why do we need this here? `project-access` is a low level module to be used by others modules to simplify project access, so why would we have any translatable strings in here?

Reply:
Resolving this question because it is discussed at https://github.com/SAP/open-ux-tools/pull/685#discussion_r975145025